tests/test_load_current_adrs.py

```diff
@@ -0,0 +1,370 @@
+
+@pytest.mark.parametrize("test_input,case_id,entity_id,expected", [
+    ("431", 1, 2,    # With reclassification
+        [{'text': '431',
+          'title': '2',
+          'type': 'statute',
+          'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html'
+          '&title=52&section=30101'}]),
+    ("30116", 1, 2,  # Already reclassified
+        [{'text': '30116',
+          'title': '52',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=52&section=30116'}]),
+    ("434(a)(11)", 1, 2,
+        [{'text': '434(a)(11)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=52&section=30104'}]),
+    ("9999", 1, 2,
+        [{'text': '9999',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9999'}]),
+    ("9993(c)(2)", 1, 2,
+        [{'text': '9993(c)(2)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9993'}]),
+    ("9993(a)(4) formerly 438(a)(4)", 1, 2,
+        [{'text': '9993(a)(4)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9993'}]),
+    ("9116(a)(2)(A), 9114(b) (formerly 441a(a)(2)(A), 434(b)), 30116(f) (formerly 441a(f))", 1, 2,
+        [{'text': '9116(a)(2)(A)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9116'},
+        {'text': '9114(b)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9114'},
+        {'text': '30116(f)',
+          'title': '52',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=52&section=30116'}]),
+    ("9993(a)(4) (formerly 438(a)(4)", 1, 2,  # No matching ')' for (formerly
+        [{'text': '9993(a)(4)',
+          'title': '2',
+          'type': 'statute',
+        'url': 'https://api.fdsys.gov/link?collection=uscode&year=mostrecent&link-type=html&title=2&section=9993'}]),
+])
+def test_parse_statutory_citations(test_input, case_id, entity_id, expected):
+    assert parse_statutory_citations(test_input, case_id, entity_id) == expected
+
+def assert_es_index_call(call_args, expected_adr):
+    index, doc_type, adr = call_args[0]
+    assert index == 'docs'
+    assert doc_type == 'adrs'
+    assert adr == expected_adr
+
+class TestLoadCurrentADRs(BaseTestCase):
```

Comment:
This test should only be testing requirements that are _unique_ to ADRs. As mentioned in other places, the common functions would be _reused_.